content/docs/user-guide/assets/asset-processor/interface.md

```diff
@@ -47,33 +46,44 @@ Jobs in the Asset Status list display the statuses below:
+* {{< icon "error.svg" >}} **Failed** - The job has failed and has emitted a failure.
 
-It's possible for a job to emit a combination of warnings and errors.
+It's possible for a job to emit a combination of warnings, errors, and failures.
 
 Completed with warnings produces a valid product asset, but might have some data substituted. Suppose a job is processing a `.fbx` file that produces several mesh product assets, and that a `.assetinfo` sidecar file specifies to process the meshes with custom normals. If the meshes in the `.fbx` file do not have custom normals, the Asset Builder generates normals instead. Valid product assets are produced, but several warnings are emitted because custom normals could not be found for the mesh assets.
 
-Completed with errors produces a valid product asset, but has incomplete data. Errors happen when data required for the product asset is missing or cannot be produced. Using the example above, suppose the `.assetinfo` specifies tangents should be generated for the mesh product assets, but the Asset Builder fails to produce the tangents. The end result is a valid product asset that does not contain all the requested data.
+Completed with errors produces a product asset but it may not be valid for all use cases, such as having incomplete data. Errors happen when data required for the product asset is missing or cannot be produced. Using the example above, suppose the `.assetinfo` specifies tangents should be generated for the mesh product assets, but the Asset Builder fails to produce the tangents. The end result is a valid product asset that does not contain all the requested data.
 
-Failed with errors does not produce a product asset.
+Failed jobs do not produce a product asset.
+
+What is considered a log message, warning, error, and failure, and how the system handles these is up to each builder's author. Our guidance is:
+ * Print a log message for anything that will be useful as a bread crumb for tracking down problems later, for both the content creator or the builder author.
+ * Emit a warning if something is wrong but it can mostly be handled, and the output your builder can generate is reasonably close to what the content creator intended.
+ * Post an error if your builder encounters data it can't properly handle and you know the product asset you output won't be fully correct. When possible, this is prefered over a failure because the failure won't produce any product asset, so any references from other assets to this asset may break. For example, if a prefab references a mesh product asset from an FBX file, an update to the FBX causes the job to fail, then any edits to the prefab while this job is failing may lose the reference to the mesh.
+ * Post a failure if your builder encounters something it cannot happen and all processing should end, and the builder should not output anything. This is generally considered a last resort, and is most commonly seen when a builder crashes during processing.

```

Comment:
I think you have a typo here "something it cannot happen". Should that be handle?

Reply:
Fixed